woocommerce/payment-gateway/assets/js/frontend/sv-wc-payment-gateway-payment-form.coffee

```diff
@@ -299,10 +299,11 @@ jQuery( document ).ready ($) ->
 			csc_required_for_tokens  = @csc_required_for_tokens
 
 			$new_payment_method_selection = $( "div.js-wc-#{ id_dasherized }-new-payment-method-form" )
-			$csc_field = $new_payment_method_selection.find( '.js-sv-wc-payment-gateway-credit-card-form-csc' ).parent()
+			$csc_field = $new_payment_method_selection.find( '.js-sv-wc-payment-gateway-credit-card-form-csc' ).parents( '.form-row' )
```

Comment:
Consider using `closest()` instead of `parents()`, as it will only match the first ancestor, while the latter will match all, which 1) could cause unintended effects on borked HTML layouts 2) be a tad slower

See https://stackoverflow.com/a/10499468